test/uk/gov/hmrc/cataloguefrontend/healthindicators/HealthIndicatorsConnectorSpec.scala

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cataloguefrontend.healthindicators
+
+import com.github.tomakehurst.wiremock.http.RequestMethod.GET
+import org.scalatest.OptionValues
+import org.scalatest.concurrent.ScalaFutures.convertScalaFuture
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+import org.scalatestplus.play.guice.GuiceOneAppPerSuite
+import play.api.Application
+import play.api.inject.guice.GuiceApplicationBuilder
+import uk.gov.hmrc.cataloguefrontend.WireMockEndpoints
+import uk.gov.hmrc.http.HeaderCarrier
+
+class HealthIndicatorsConnectorSpec
+  extends AnyWordSpec
+    with Matchers
+    with GuiceOneAppPerSuite
+    with WireMockEndpoints
+    with OptionValues {
+
+  private implicit val headerCarrier: HeaderCarrier = HeaderCarrier()
+  override def fakeApplication: Application =
```

Comment:
I think creating this `fakeApplication` is duplicated in other connector specs as well. Can we extract this into a common trait and mixin that trait?

Reply:
Same for controller specs

Reply:
So it looks like I would have to override the configuration for each different connector, so it may not be worth doing? WDYT?